lib/cc/analyzer/issue_validator.rb

```diff
@@ -0,0 +1,49 @@
+module CC
+  module Analyzer
+    class IssueValidator
+      CHECKS = [
+        IssueCategoryValidation,
+        IssueCheckNamePresenceValidation,
+        IssueDescriptionPresenceValidation,
+        IssuePathExistenceValidation,
```

Comment:
There's an interesting ordering issue here that might prompt us to not sort all of these alphabetically: checking for path presence *after* checking for path existence seems wrong to me.

I think either we should re-order these to `presence -> relative -> existence` (and add a comment explaining why), or collapse all of these into a single `IssuePathValidation`.